pytorch_lightning/loops/base.py

```diff
@@ -46,6 +52,15 @@ class Loop(ABC):
     def __init__(self) -> None:
         self.iteration_count: int = 0
         self.trainer: Optional['pl.Trainer'] = None
+        self._is_restarting = False
+
+    @property
+    def is_restarting(self) -> bool:
```

Comment:
Should the name be `restarting` to be consistent with `done`? It's not `is_done`

Reply:
Done !